reV/supply_curve/point_summary.py

```diff
@@ -206,8 +215,29 @@ def mean_lcoe(self):
         mean_lcoe : float | None
             Mean LCOE value for the non-excluded data.
         """
+
         mean_lcoe = None
-        if self.lcoe_data is not None:
+
+        # prioritize the calculation of lcoe explicitly
+        # from the multi year mean CF
```

Comment:
This doesn't necessarily have to be the MY mean CF, maybe just say from the mean CF?

Reply:
we can but there would be no point to re-calculating the LCOE from a single year mean, so i think this comment is appropriate in describing this code block

Reply:
Okay thats what I thought, but I'm still a bit confused, how does the code know that its MY Means and to re-compute? If its doesn't maybe we should just clarify that this code will prioritize the re-calculation of lcoe explicitly in case the input data is a Multi-year mean?

Reply:
The code doesnt know if its multi year CF or not, but if its single year CF then the re-calculated LCOE should match / not matter. the code here is really of consequence for multi year. Another reason we shouldn't raise errors or force the user into anything.

Reply:
Seeing as this still confuses the hell out of me I still think
1) making the comment clearer, even jus to stat the above would be helpful
2)Somewhere to raise a warning that LCOE can't re recomputed and that if you are running MY mean CF your results might be off, I think you do that below (third comment), but just wanted to make sure.